modules/aws-load-balancer-operator-prereqs.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * networking/aws-load-balancer.adoc
+
+:_content-type: CONCEPT
+[id="aws-load-balancer-operator-prereqs_{context}"]
+= Prerequisites for configuring an AWS Load Balancer Operator
+
+.Prerequisites
+* You have created a multi-AZ ROSA cluster deployed with STS
+* {product-title} (ROSA)CLI
+* OC CLI
```

Comment:
```suggestion
* You have created a multi-AZ ROSA cluster deployed with STS.
* You have installed the {product-title} (ROSA) CLI.
* You have installed the OpenShift CLI (`oc`).
```


Per [the ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#wording-of-list-items), the list items need to be grammatically parallel